frontend/src/actions/failures/delete/index.js

```diff
@@ -0,0 +1,75 @@
+import API, { parseResponseError } from 'api'
+import { addFlashMessage } from 'actions/flash-messages'
+
+import {
+  FAILURE_SHOW_DELETE,
+  FAILURE_HIDE_DELETE,
+  REQUEST_FAILURE_DELETE,
+  RECEIVE_FAILURE_DELETE,
+  REQUEST_FAILURE_DELETE_FAILED,
+  FAILURE_HIDE_OVERVIEW,
+  DELETE_FAILURE
+} from 'actions'
+
+export const showFailureDelete = (failure) => ({
+  type: FAILURE_SHOW_DELETE,
+  failure
+})
+
+export const hideFailureDelete = (failure) => ({
+  type: FAILURE_HIDE_DELETE,
+  failure
+})
+
+const requestFailureDelete = (failure) => ({
+  type: REQUEST_FAILURE_DELETE,
+  failure
+})
+
+const receiveFailureDelete = (failure, data) => ({
+  type: RECEIVE_FAILURE_DELETE,
+  acknowledged: data.acknowledged,
+  failure
+})
+
+const hideFailureOverview = (failure) => ({
```

Comment:
This event generator is defined here and in `/frontend/src/actions/failures/overview.js`
This can leave to weird problems in the future.
I don't know what's the standard solution. Maybe importing the action from the other place is good enough. Thoughts?